bindings/python/src/CrashDumpBindings.cpp

```
@@ -0,0 +1,255 @@
+                syncExtraToNative(self, dump);
+            },
+            py::arg("key"),
+            py::arg("value"))
+        .def(
+            "__contains__", [](py::object self, const std::string& key) -> bool { return getExtraDict(self).contains(key); }, py::arg("key"))
+        .def(
+            "__delitem__",
+            [](py::object self, const std::string& key) {
+                auto d = getExtraDict(self);
+                if(!d.contains(key)) throw py::key_error(key);
+                PyDict_DelItemString(d.ptr(), key.c_str());
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)